modules/nw-ingress-gateway-api-manage-succession.adoc

```diff
@@ -0,0 +1,37 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-manage-succession_{context}"]
+= Preparing for Gateway API management succession by the Ingress Operator
+
+In {product-title} {product-version}, the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions (CRDs). Upgrading from a previous version of {product-title} where this management was not present requires you to make changes or remove any Gateway API CRDs that already exist in the cluster so that they conform to the specific {product-title} specification required by the Ingress Operator.
```

Comment:
```suggestion
Starting in {product-title} 4.19, the Ingress Operator manages the lifecycle of any Gateway API custom resource definitions (CRDs). Upgrading from a version prior to 4.19 of {product-title} where this management was not present requires you to replace or remove any Gateway API CRDs that already exist in the cluster so that they conform to the specific {product-title} specification required by the Ingress Operator.  {product-title} version 4.19 requires Gateway API Standard v1.2.0 CRDs.
```